src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx

```diff
@@ -54,10 +57,16 @@ const ColorPickerItemComponent = ({
       >
         {shouldRenderIndicatorWithoutBackground ? (
```

Comment:
The diff here is the style - maybe just add conditional style instead of the if (small refactor to the old code but it will look much better)